unixHttpRoutes.go

```diff
@@ -83,6 +83,10 @@ func AddProcessRoutes(router *httprouter.Router) {
 	router.GET("/net/events", handleSubscribeNetworkEvents)
 	router.GET("/net/events/:subscription", handleGetLatestNetworkEvents)
 
+	router.GET("/dns", handleGetDNS)
```

Comment:
would "/net/dns" be a more consistent path?

Reply:
done